src/black/strings.py

```diff
@@ -236,3 +247,31 @@ def normalize_string_quotes(s: str) -> str:
         return s  # Prefer double quotes
 
     return f"{prefix}{new_quote}{new_body}{new_quote}"
+
+
+def normalize_unicode_escape_sequences(leaf: Leaf) -> None:
+    """Replace hex codes in Unicode escape sequences with lowercase representation."""
+    text = leaf.value
+    prefix = get_string_prefix(text)
```

Comment:
why not return early if this contains a `r` ?

```python
if "r" in get_string_prefix(text).lower():
    return
```

Reply:
Done